api/envoy/config/listener/v3/udp_listener_config.proto

```diff
@@ -33,8 +33,18 @@ message UdpListenerConfig {
 
   // Configuration for QUIC protocol. If empty, QUIC will not be enabled on this listener. Set
   // to the default object to enable QUIC without modifying any additional options.
-  // [#not-implemented-hide:]
-  // [#comment:Unhide when QUIC alpha is announced with other docs.]
+  //
+  // .. note::
+  //   QUIC support is currently alpha and should be used with caution.
```

Comment:
im wondering if this note should be a warning - and also thinking it might be better placed in docs/root/intro/arch_overview/http/http3.rst as its linked to from various places - feels a bit arbitrary to add it to one proto

Reply:
im wondering if this should make use of the `experimental` annotation that has just been added

agreed that their needs to be a warning here - but i would put the fuller explanation in the ref docs and point to that - as is done in most of the the api protos - mho